### ISSUE TITLE
Training loss is Not decreasing and saturates around ~-5, -6 when training with Arabic data set

### ISSUE THREAD

**Issue by FadyKhalaf** [NVIDIA/waveglow#173]
Hello, 

my configuration parameters are as follows
{
    "train_config": {
        "output_directory": "checkpoints",
        "epochs": 100000,
        "learning_rate": 1e-5,
        "sigma": 1.0,
        "iters_per_checkpoint": 2000,
        "batch_size": 2,
        "seed": 1234,
        "checkpoint_path": ""
    },
    "data_config": {
        "training_files":"train_files.txt",
        "segment_length": 16000,
        "sampling_rate": 22050,
        "filter_length": 1024,
        "hop_length": 256,
        "win_length": 1024,
        "mel_fmin": 0.0,
        "mel_fmax": 8000.0
    },
    "dist_config": {
        "dist_backend": "nccl",
        "dist_url": "tcp://localhost:54321"
    },

    "waveglow_config": {
        "n_mel_channels": 80,
        "n_flows": 12,
        "n_group": 8,
        "n_early_every": 4,
        "n_early_size": 2,
        "WN_config": {
            "n_layers": 8,
            "n_channels": 512,
            "kernel_size": 3
        }
    }
}

I am training on 1 GPU NVIDIA V100 with batch size of 2 samples, currently at epoch 60 and yet no sign to a steady decrease in reduced loss.

any advice or Tip?

**Comment by rafaelvalle** [comment]
Did you try using the pre-trained waveglow?
